include/utils/MastodonUtils.h

```diff
@@ -18,7 +18,7 @@
 // MOOSE includes
 #include "GeneralVectorPostprocessor.h"
 // Other includes
-#include <boost/math/distributions.hpp>
+#include "BoostDistribution.h"
```

Comment:
I think that this can just be removed. Also the `boost::math::quantile` and similar methods should be changed to be `T.quantile()`, etc.

Reply:
I kept BoostDistribution.h since I am using Boost distributions in the greaterProbability function template in MastodonUtils.h.